source/state_machines_macros/src/to_token_stream.rs

```diff
@@ -301,6 +301,7 @@ pub fn output_primary_stuff(
     let attrs = &bundle.sm.attrs;
     let code: TokenStream = quote_spanned! { sm.fields_named_ast.span() =>
         #[cfg_attr(verus_keep_ghost, verus::internal(verus_macro))]
+        #[verifier::ext_equal]
```

Comment:
You might want to make this `#[cfg_attr(verus_keep_ghost, verifier::ext_equal)]`.